README.md

```diff
@@ -73,7 +73,7 @@ For local use, the `.scripts/run_config.sh` script can be used to run a config -
 For those who wish to use other tools, one may configure playbooks and config JSON files for the desired model, then use the `.scripts/run_configs.sh` script's `-t` and `-o` flags to fill templates and output pure SQL to file:
 
 ```bash
-bash .scripts/run_config.sh -b ~/pathTo/sql-runner -c web/v1/bigquery/configs/example_with_custom.json -t -o tmp/sql;
+bash .scripts/run_config.sh -b ~/pathTo/sql-runner -c web/v1/bigquery/sql-runner/configs/example_with_custom.json -t .scripts/templates/bigquery.yml.tmpl -o tmp/sql;
```

Comment:
Same as the scripts readme here and in the above paragraph  - I meant `-p` instead of `-t`